src/vm/functions/conversions.rs

```diff
@@ -113,3 +115,107 @@ pub fn native_buff_to_uint_be(value: Value) -> Result<Value> {
     }
     return buff_to_int_generic(value, EndianDirection::BigEndian, convert_to_uint_be);
 }
+
+pub fn native_string_to_int_generic(
+    value: Value,
+    conversion_fn: fn(String) -> Result<Value>,
+) -> Result<Value> {
+    match value {
+        Value::Sequence(SequenceData::String(CharType::ASCII(ASCIIData { data }))) => {
+            let as_string = String::from_utf8(data).unwrap();
```

Comment:
This is a runtime crash waiting to happen.  There are exactly zero circumstances where a `.unwrap()` or `.expect()` is acceptable when processing user-submitted input.

Reply:
good catch! I believe I have removed all of the "unwrap" instances from the production code in this change.